beta-src/src/interfaces/PhaseScroll.ts

```diff
@@ -0,0 +1,7 @@
+import { ScrollButtonState } from "../enums/UIState";
+
+export interface scrollButtonProps {
+  onClick?: React.MouseEventHandler<HTMLButtonElement> | undefined;
+  direction: ScrollButtonState;
+  disabled?: boolean;
```

Comment:
alpha order props...i think it's better to have the props inside the component so that when people look at the component they can see them in the same context.